### DISCUSSION TITLE
有计划上传到微软的商店吗

### DISCUSSION THREAD

**Discussion by loveqianool** [baptistecdr/aria2-extensions#29] [General]
虽然可以在 chrome 的扩展商店下载，但是在那边下载的扩展不能同步，挺难受。

**Comment by baptistecdr** [comment]
Hello,

Is it possible de write in English please ?

Sincerely,
baptistecdr

**Reply by loveqianool** [reply]
Is there any plan to put it on the Microsoft Edge store ?

**Reply by baptistecdr** [reply]
Yes it is planned and I already tried. I need to write a privacy policy or something like that which I did but it didn't pass the validation.

**Reply by baptistecdr** [reply]
Done https://microsoftedge.microsoft.com/addons/detail/aria2-integration/hmmpdilndjfmceolhbgjejogjaglbiel
